### PR TITLE
Delete: chrdavis.smartrename version 1.0.4 (version conflict)

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/21741)

### CODE REVIEW

File: manifests/c/chrdavis/smartrename/1.0.4/chrdavis.smartrename.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+PackageIdentifier: chrdavis.smartrename
+PackageVersion: 1.0.4
+PackageLocale: en-US
+Publisher: chrdavis
+PublisherUrl: https://github.com/chrdavis
+PublisherSupportUrl: https://github.com/chrdavis/SmartRename/issues
+#PrivacyUrl: 
+Author: Chris Davis
+PackageName: Smart Rename
```
Comment (ItzLevvie) [inline manifests/c/chrdavis/smartrename/1.0.4/chrdavis.smartrename.locale.en-US.yaml:11]:
```suggestion
PackageName: SmartRename
```

File: manifests/c/chrdavis/smartrename/1.0.4/chrdavis.smartrename.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,32 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: chrdavis.smartrename
+PackageVersion: 1.0.4
+PackageLocale: en-US
+Publisher: chrdavis
```
Comment (ItzLevvie) [inline manifests/c/chrdavis/smartrename/1.0.4/chrdavis.smartrename.locale.en-US.yaml:6]:
```suggestion
Publisher: Chris Davis
```

File: manifests/c/chrdavis/smartrename/1.0.4/chrdavis.smartrename.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,32 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: chrdavis.smartrename
+PackageVersion: 1.0.4
```
Comment (ItzLevvie) [inline manifests/c/chrdavis/smartrename/1.0.4/chrdavis.smartrename.locale.en-US.yaml:4]:
```suggestion
PackageVersion: 1.0.0
```

File: manifests/c/chrdavis/smartrename/1.0.4/chrdavis.smartrename.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,21 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: chrdavis.smartrename
+PackageVersion: 1.0.4
```
Comment (ItzLevvie) [inline manifests/c/chrdavis/smartrename/1.0.4/chrdavis.smartrename.installer.yaml:4]:
```suggestion
PackageVersion: 1.0.0
```

File: manifests/c/chrdavis/smartrename/1.0.4/chrdavis.smartrename.yaml [hunk 5]
```diff
@@ -1,21 +1,8 @@
...
-Author: chrdavis
-ShortDescription: A Windows Shell Extension for more advanced bulk renaming using search and replace or regular expressions
-Tags:
-- file
-- rename
-- shell
-PackageUrl: https://github.com/chrdavis/SmartRename
-License: MIT
-LicenseUrl: https://github.com/chrdavis/SmartRename/blob/master/LICENSE
 PackageVersion: 1.0.4
```
Comment (ItzLevvie) [inline manifests/c/chrdavis/smartrename/1.0.4/chrdavis.smartrename.yaml:4]:
```suggestion
PackageVersion: 1.0.0
```

File: manifests/c/chrdavis/smartrename/1.0.4/chrdavis.smartrename.installer.yaml [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+- silent
+- silentWithProgress
+Installers:
+- InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: msi
+  Scope: machine
+  InstallerUrl: https://github.com/chrdavis/SmartRename/releases/download/v.1.0.4/SmartRenameInstaller64.msi
+  InstallerSha256: 47FE5A7D9916C96B0E148E706184BC93BB2511B0344BF21F12BAE79907EA2F70
+#  ProductCode:
```
Comment (ItzLevvie) [inline manifests/c/chrdavis/smartrename/1.0.4/chrdavis.smartrename.installer.yaml:17]:
```suggestion
  ProductCode: "{688CD75D-0CEE-4E10-8552-7AE64CA687C7}"
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Comment by vedantmgoyal2009** [comment]
I have committed changes and they are causing merge conflicts. Should I roll back?

**Comment by ItzLevvie** [comment]
No because 1.0.4 isn't what's shown in Control Panel; it seems like you'll have to update the existing manifest instead.

**Comment by ItzLevvie** [comment]
Install all of the versions in a VM and double check the version, if all of the versions show as `1.0.0` then delete all of the manifests and change the `1.0.0` manifest InstallerUrl to the latest version of the application.

**Comment by vedantmgoyal2009** [comment]
Ok, I am creating a PR to delete the original version 1.0.0 so that this PR won't create merge conflicts.

**Comment by vedantmgoyal2009** [comment]
I am deleting all files in this PR as the latest version is already added.

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**